packages/manager/modules/adp/src/deploy/storage.html

```diff
@@ -0,0 +1,112 @@
+<div data-ng-if="$ctrl.storage.initialized">
+    <div class="row">
+        <div class="col-sm-6 col-xs-12 mb-5">
+            <oui-field data-label="{{ ::'adp_deploy_storage_cluster_label' | translate }}"
+                data-help-text="{{ ::'adp_deploy_storage_cluster_storage_helptext' | translate: {
+                    adp_version: $ctrl.adp.cluster_type,
+                    min_storage: $ctrl.storage.min_cluster_storage,
+                    max_storage: $ctrl.storage.max_cluster_storage
+                } }}"
+                data-size="m"
+                data-error-messages="{ pattern: ('adp_deploy_storage_invalid_entry' | translate) }">
+                <div class="oui-input-group">
+                    <input class="oui-input oui-input_number"
+                        type="number"
+                        id="hdfs_effective_storage"
+                        name="hdfs_effective_storage"
+                        required
+                        placeholder="{{ ::'adp_deploy_storage_set_quantity' | translate }}"
+                        data-ng-min="{{ ::$ctrl.storage.min_cluster_storage }}"
+                        data-ng-max="{{ ::$ctrl.storage.max_cluster_storage }}"
+                        data-ng-pattern="/^\d+$/"
+                        data-ng-model="$ctrl.storage.hdfs_effective_storage">
+                    <button class="oui-button oui-button_icon-only oui-button_small-width"
+                        type="button"
+                        disabled>
+                        <span data-translate="adp_common_unit_gb"></span>
```

Comment:
You can apply `translate` directive directly on the parent element and get rid off that `span` element.

Reply:
Done